{"uid":"f98678aa8b0cfc44","name":"Test get all entities","fullName":"tests.api.test_api.TestEntityAPI#test_get_all_entities","historyId":"9d79d9e140415fb9f4a303addeb457db","time":{"start":1727257985828,"stop":1727257985863,"duration":35},"description":"Verify that all entities can be retrieved","descriptionHtml":"<p>Verify that all entities can be retrieved</p>\n","status":"broken","statusMessage":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll","statusTrace":"self = <api.test_api.TestEntityAPI object at 0x7fa1c7ed90a0>\napi_client = <services.entity.api_client.APIClient object at 0x7fa1c7ee33e0>\n\n    @allure.title(\"Test get all entities\")\n    @allure.description(\"Verify that all entities can be retrieved\")\n    @pytest.mark.api\n    def test_get_all_entities(self, api_client: APIClient) -> None:\n        \"\"\"Tests that all entities can be retrieved successfully.\"\"\"\n>       entities = api_client.get_all_entities()\n\ntests/api/test_api.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/entity/api_client.py:43: in get_all_entities\n    response.raise_for_status()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Response [404]>\n\n    def raise_for_status(self):\n        \"\"\"Raises :class:`HTTPError`, if one occurred.\"\"\"\n    \n        http_error_msg = \"\"\n        if isinstance(self.reason, bytes):\n            # We attempt to decode utf-8 first because some servers\n            # choose to localize their reason strings. If the string\n            # isn't utf-8, we fall back to iso-8859-1 for all other\n            # encodings. (See PR #3538)\n            try:\n                reason = self.reason.decode(\"utf-8\")\n            except UnicodeDecodeError:\n                reason = self.reason.decode(\"iso-8859-1\")\n        else:\n            reason = self.reason\n    \n        if 400 <= self.status_code < 500:\n            http_error_msg = (\n                f\"{self.status_code} Client Error: {reason} for url: {self.url}\"\n            )\n    \n        elif 500 <= self.status_code < 600:\n            http_error_msg = (\n                f\"{self.status_code} Server Error: {reason} for url: {self.url}\"\n            )\n    \n        if http_error_msg:\n>           raise HTTPError(http_error_msg, response=self)\nE           requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll\n\n/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/requests/models.py:1024: HTTPError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"api_client","time":{"start":1727257985828,"stop":1727257985828,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_skip_sensitive","time":{"start":1727257985828,"stop":1727257985828,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Verify that all entities can be retrieved","status":"broken","statusMessage":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll","statusTrace":"self = <api.test_api.TestEntityAPI object at 0x7fa1c7ed90a0>\napi_client = <services.entity.api_client.APIClient object at 0x7fa1c7ee33e0>\n\n    @allure.title(\"Test get all entities\")\n    @allure.description(\"Verify that all entities can be retrieved\")\n    @pytest.mark.api\n    def test_get_all_entities(self, api_client: APIClient) -> None:\n        \"\"\"Tests that all entities can be retrieved successfully.\"\"\"\n>       entities = api_client.get_all_entities()\n\ntests/api/test_api.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/entity/api_client.py:43: in get_all_entities\n    response.raise_for_status()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Response [404]>\n\n    def raise_for_status(self):\n        \"\"\"Raises :class:`HTTPError`, if one occurred.\"\"\"\n    \n        http_error_msg = \"\"\n        if isinstance(self.reason, bytes):\n            # We attempt to decode utf-8 first because some servers\n            # choose to localize their reason strings. If the string\n            # isn't utf-8, we fall back to iso-8859-1 for all other\n            # encodings. (See PR #3538)\n            try:\n                reason = self.reason.decode(\"utf-8\")\n            except UnicodeDecodeError:\n                reason = self.reason.decode(\"iso-8859-1\")\n        else:\n            reason = self.reason\n    \n        if 400 <= self.status_code < 500:\n            http_error_msg = (\n                f\"{self.status_code} Client Error: {reason} for url: {self.url}\"\n            )\n    \n        elif 500 <= self.status_code < 600:\n            http_error_msg = (\n                f\"{self.status_code} Server Error: {reason} for url: {self.url}\"\n            )\n    \n        if http_error_msg:\n>           raise HTTPError(http_error_msg, response=self)\nE           requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll\n\n/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/requests/models.py:1024: HTTPError","steps":[{"name":"Get all entities","time":{"start":1727257985829,"stop":1727257985863,"duration":34},"status":"broken","statusMessage":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/TestNest/TestNest/services/entity/api_client.py\", line 43, in get_all_entities\n    response.raise_for_status()\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/requests/models.py\", line 1024, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","steps":[],"attachments":[{"uid":"13957dd40252fb96","name":"API Response","source":"13957dd40252fb96.txt","type":"text/plain","size":2430},{"uid":"39b12d7b58c71dcb","name":"Status Code","source":"39b12d7b58c71dcb.txt","type":"text/plain","size":3},{"uid":"44758c5209c4fa44","name":"Headers","source":"44758c5209c4fa44.txt","type":"text/plain","size":197}],"parameters":[{"name":"title","value":"None"},{"name":"verified","value":"None"},{"name":"page","value":"None"},{"name":"per_page","value":"None"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":3,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"f66f8bccad43b887","name":"log","source":"f66f8bccad43b887.txt","type":"text/plain","size":1565},{"uid":"be9191d8f2dad3ba","name":"stderr","source":"be9191d8f2dad3ba.txt","type":"text/plain","size":1248}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":5,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Entity Management"},{"name":"feature","value":"CRUD Operations"},{"name":"tag","value":"api"},{"name":"parentSuite","value":"tests.api"},{"name":"suite","value":"test_api"},{"name":"subSuite","value":"TestEntityAPI"},{"name":"host","value":"fv-az1490-494"},{"name":"thread","value":"2253-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.api.test_api"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"ddda5721b43f24b9","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll","time":{"start":1727257980777,"stop":1727257980813,"duration":36}},{"uid":"3c80d2e17a4eedb6","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll","time":{"start":1727257975727,"stop":1727257975762,"duration":35}},{"uid":"c985c368ee30be4f","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll","time":{"start":1727257970643,"stop":1727257970711,"duration":68}},{"uid":"c16aa9eb3974965c","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll","time":{"start":1727257965593,"stop":1727257965627,"duration":34}},{"uid":"8ef20206e8b63d7e","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/getAll","time":{"start":1727257960543,"stop":1727257960577,"duration":34}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":3,"unknown":0,"total":4},"items":[{"uid":"47ef091a208f0eb","reportUrl":"https://ZhikharevAl.github.io/TestNest/7//#testresult/47ef091a208f0eb","status":"passed","time":{"start":1727235456697,"stop":1727235457135,"duration":438}},{"uid":"ff24ff0db6d8528a","reportUrl":"https://ZhikharevAl.github.io/TestNest/6//#testresult/ff24ff0db6d8528a","status":"passed","time":{"start":1727234277086,"stop":1727234277476,"duration":390}},{"uid":"e5220fa9e065f7e6","reportUrl":"https://ZhikharevAl.github.io/TestNest/5//#testresult/e5220fa9e065f7e6","status":"passed","time":{"start":1727230687451,"stop":1727230687809,"duration":358}}]},"tags":["api"]},"source":"f98678aa8b0cfc44.json","parameterValues":[]}